### ISSUE TITLE
Gatsby build fails with "ReactDOMServer does not yet support Suspense."

### ISSUE THREAD

**Issue by jacqui** [news-catalyst/tiny-news-demo#23]
It looks like one of the embed options is using a library that gatsby doesn't work with. I'm investigating. I noticed the deploy was failing on `gatsby build` and was able to replicate the issue running that command locally.

```
success Building production JavaScript and CSS bundles - 21.876s
success run queries - 23.199s - 23/23 0.99/s
[============================]   4.085 s 24/24 100% Generating image thumbnails
[                            ]   0.001 s 0/26 0% Building static HTML for pages
found match:  https://www.youtube.com/embed/LLpIMRowndg
found match:  https://twitter.com/BBCWorld/status/1262182320209092609
found match:  [tweet id=1262182320209092609]
failed Building static HTML for pages - 4.984s

 ERROR #95313 

Building static HTML failed for path "/articles/article-with-embeds"

See our docs page for more info on this error: https://gatsby.dev/debug-html



  Error: Minified React error #294; visit https://reactjs.org/docs/error-decoder.html?invariant=294 for the full message or use the non-minified dev environment for full errors and additional helpful warn  ings.
  
  - react-dom-server.node.production.min.js:46 a.b.render
    [tinynewsdemo]/[react-dom]/cjs/react-dom-server.node.production.min.js:46:17
  
  - react-dom-server.node.production.min.js:44 a.b.read
    [tinynewsdemo]/[react-dom]/cjs/react-dom-server.node.production.min.js:44:18
  
  - react-dom-server.node.production.min.js:54 renderToString
    [tinynewsdemo]/[react-dom]/cjs/react-dom-server.node.production.min.js:54:364
  
  - render-page.js:640 Module../.cache/static-entry.js.__webpack_exports__.default
    /Users/jacqui/Projects/newscatalyst/tinynewsdemo/public/render-page.js:640:28
  
  - render-html.js:28 
    [tinynewsdemo]/[gatsby]/dist/utils/worker/render-html.js:28:36
  
  - debuggability.js:384 Promise._execute
    [tinynewsdemo]/[bluebird]/js/release/debuggability.js:384:9
  
  - promise.js:518 Promise._resolveFromExecutor
    [tinynewsdemo]/[bluebird]/js/release/promise.js:518:18
  
  - promise.js:103 new Promise
    [tinynewsdemo]/[bluebird]/js/release/promise.js:103:10
  
  - render-html.js:24 
    [tinynewsdemo]/[gatsby]/dist/utils/worker/render-html.js:24:47
  
  - util.js:16 tryCatcher
    [tinynewsdemo]/[bluebird]/js/release/util.js:16:23
  
  - map.js:68 MappingPromiseArray._promiseFulfilled
    [tinynewsdemo]/[bluebird]/js/release/map.js:68:38
  
  - promise_array.js:115 MappingPromiseArray.PromiseArray._iterate
    [tinynewsdemo]/[bluebird]/js/release/promise_array.js:115:31
  
  - promise_array.js:79 MappingPromiseArray.init
    [tinynewsdemo]/[bluebird]/js/release/promise_array.js:79:10
  
  - map.js:37 MappingPromiseArray._asyncInit
    [tinynewsdemo]/[bluebird]/js/release/map.js:37:10
  
  - async.js:97 _drainQueueStep
    [tinynewsdemo]/[bluebird]/js/release/async.js:97:12
  
  - async.js:86 _drainQueue
    [tinynewsdemo]/[bluebird]/js/release/async.js:86:9
  
```

**Comment by jacqui** [comment]
Looks like the issue was in `react-player`, a dependency of `react-embed`, and was fixed in react-player v2.0.1

the embed library has been updated, but not released, to include the latest version of the player: https://github.com/streamich/react-embed/pull/303

I'm trying to require a newer version of the player now to get our builds back in action.

More info https://github.com/CookPete/react-player/issues/738

**Comment by jacqui** [comment]
Fixed in PR #24
